inference-engine/src/vpu/graph_transformer/src/middleend/passes/eliminate_redundant_conversions.cpp

```diff
@@ -29,6 +29,13 @@ void PassImpl::runForStage(const Model& model, const Stage& convert) {
     const auto input = convert->input(0);
     const auto output = convert->output(0);
 
+    // Remove converts added to unused inputs
```

Comment:
Add an example of case when such unused inputs appear. BTW, you say about inputs, but check on `Intermediate` data objects?

Reply:
Simple example would be `Parameter->Shapeof->Result` which is constant folded to `Parameter, Const->Result` (in the case from #-39883 we have multiple such inputs).
I check on intermediate because if we have previously added the `Convert` stage to unused input, we have unused `Intermediate` data object afterwards.

Reply:
So fix the comment in the code. It'd would be nice to have explanation there about input -> convert -> intermediate and so on

Reply:
Extended the comment. Feel free to suggest something if it's still unclear.